flixel/math/FlxPoint.hx

```diff
@@ -152,6 +152,19 @@ class FlxPoint implements IFlxPooled
 	}
 	
 	/**
+	 * Scale this point.
+	 * 
+	 * @param	k - scale coefficient
+	 * @return	scaled vector
+	 */
+	public inline function scale(k:Float):FlxVector
```

Comment:
Hm, this is going to result in `sprite.scale.scale()`, which looks quite weird... Oh well.